packages/ui-components/src/Input/EmailPasswordInput.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from "react";
+import { BaseTextInput, AllProps } from "./TextTypeInput";
+import "./EmailPassword.module.scss";
+import { Icon } from "../Icon/Icon";
+
+interface Validator {
+  label: string;
+  fn: (value: string) => boolean;
+}
+
+export type ExistingPasswordProps = Omit<AllProps, "multiline">;
+export type NewPasswordProps = Omit<
+  ExistingPasswordProps & { validators: Validator[] },
+  "forgotPasswordLinkElement"
+>;
+export type EmailProps = NewPasswordProps;
+
+export enum PasswordInputType {
+  Text = "text",
+  Password = "password",
+}
+
+// types are provided here, because although the Field passes them into these
+// Input components, if Input components are used without wrapper Fields,
+// they need types
+export const EmailInput = ({ ...props }: EmailProps) => {
+  return <BaseTextInput type="email" {...props} />;
+};
+
+export const NewPasswordInput = ({
+  meta,
+  validators = undefined,
+  ...rest
+}: NewPasswordProps) => {
+  const [error, setError] = useState(meta && meta.error);
+  const [touched, setTouched] = useState(meta && meta.touched);
+
+  useEffect(() => {
+    setError(meta && meta.error);
+    setTouched(meta && meta.touched);
+  }, [meta]);
+
+  // validators is a wrapper array of validation functions
+  const validator = validators && validators.length > 0 && validators[0];
```

Comment:
we currently only use 0/1 validators, so instead of passing an array (which was only being used to get the validator label), i've just changed the props to conditionally accept the validator label.